openapi/components/paths/users.yaml

```diff
@@ -103,13 +103,36 @@ paths:
               $ref: ../requests/UpdateUserRequest.yaml
       tags:
         - users
+  'users/{userId}/groups':
+    get:
+      summary: Get User's groups
+      tags:
+        - users
+      responses:
+        '200':
+          description: OK
+          content:
```

Comment:
Linting will trigger on this once included in paths; "No direct content".
Look at `LimitedUserListResponse` on how to make a `GroupListResponse`. Having unnamed objects causes generation issues (e.g., "MissingNameObject1").